README.md

```diff
@@ -5,48 +5,57 @@
 Display CPU frequency, usage, memory consumption, and battery percentage remaining within the VSCode status bar.
 
 ## Screenshots
-![An example of resmon running](images/example.png).
 
-![Disk space feature](images/ResmonDiskSpaceScreenshot.png).
+![An example of resmon running](images/example.png).
 
+![Disk space feature](images/disk_space_screenshot.png).
```

Comment:
Where are you getting this file name? I think your naming is better, though, so maybe I should just rename the other file. 

Are you going to change the file name of the disk space screenshot in another commit?